common.yml

```diff
@@ -386,73 +367,12 @@ functions:
         chmod +x ./bin/*
         rm -rf ./mongodb-*
 
-  "fetch tool" :
```

Comment:
I removed the dependency that the qa tests have on dist. It's relatively cheap for us to build the tools in each task (only a few seconds in total), and it removes cruft from the evg project file